backend/v3/lumens.js

```diff
@@ -0,0 +1,30 @@
+import * as commonLumens from "../../common/lumens.js";
+import BigNumber from "bignumber.js";
+
+let totalSupplyData;
+let totalSupplySumData;
+
+export const totalSupplyDataHandler = function(req, res) {
```

Comment:
💡 Could we name this totalSupplyHandler? I don't think the `Data` helps communicate the intent of what this does anymore than it did without it, and it can be helpful in API upgrades if we preserve as much as possible of the old terminology for things that do not matter or do not need to change.

Reply:
https://github.com/stellar/dashboard/pull/182/commits/87afef9d9eb0fb3d97d698dcab26ff2660f2cb7a